### ISSUE TITLE
Reduce overdraw on About activity

### ISSUE THREAD

**Issue by VishnuSanal** [TeamAmaze/AmazeFileManager#3109]
`activity_about.xml` currently uses `LinearLayout`s, nested `RelativeLayout`s etc. I think we can reduce overdraws by using a `ConstraintLayout`. Moving "Translators" and "Contributors" to `RecyclerView`s would be helpful too.

**Comment by ArunPrasanth-V** [comment]
i would like to **contribute!** what i need to do now?

**Comment by VishnuSanal** [comment]
> i would like to **contribute!**

Sure!

> what i need to do now? 

As said in the comment, the purpose of this issue is essentially to reduce overdraw on `activity_about`. Moving all the nested layouts to a `ConstraintLayout` would be a good start. Making "contributors" & "translators" sections dynamic would help too (@VishalNehra should these be individual issues?)

**Comment by VishnuSanal** [comment]
> Making "contributors" & "translators" sections dynamic would help too (@VishalNehra should these be individual issues?)

@ArunPrasanth-V seems like you can decide this - can you work on both under this, provided this is a `Issue-Easy (good first issue)`? If not, feel free do the migration to `ConstraintLayout` only, we can make the other parts a new issue.

**Comment by ArunPrasanth-V** [comment]
since it is my first contribution. i need to move all the nested layout to a  **constraintLayout**.
if there any reference means that would be more helpfull either video or blog.

**Comment by VishnuSanal** [comment]
@ArunPrasanth-V Hi! Thanks for showing interest. I think [docs](https://developer.android.com/reference/androidx/constraintlayout/widget/ConstraintLayout) would be a good place to started with ConstraintLayout. Also, feel free to join [our telegram group](https://t.me/AmazeFileManager) if you've any queries :)

**Comment by ArunPrasanth-V** [comment]
@VishnuSanal i'm having doubt  As you said moving all the nested layouts such as Relative layout. whether i need to replace all the **RelativeLayout** to **ConstraintLayout**  in the [code](https://github.com/ArunPrasanth-V/AmazeFileManager/blob/release/3.7/app/src/main/res/layout/activity_about.xml).

**Comment by VishnuSanal** [comment]
@ArunPrasanth-V Nope, I felt like everything can be done with a single ConstraintLayout as the root. If we replace every RelativeLayout with ConstraintLayout, we aren't reducing any overdraw.

**Comment by ArunPrasanth-V** [comment]
@VishnuSanal I need to change root layout (LinearLayout) to ConstraintLayout right! in the given [code](https://github.com/ArunPrasanth-V/AmazeFileManager/blob/release/3.7/app/src/main/res/layout/activity_about.xml) 

1) In line number [63]( https://github.com/ArunPrasanth-V/AmazeFileManager/blob/a7da8dd623c97817f72e5a1c7c7055495ab44308/app/src/main/res/layout/activity_about.xml#L63) only
(or) i need change [ 76](https://github.com/ArunPrasanth-V/AmazeFileManager/blob/a7da8dd623c97817f72e5a1c7c7055495ab44308/app/src/main/res/layout/activity_about.xml#L76), [authors (207)](https://github.com/ArunPrasanth-V/AmazeFileManager/blob/a7da8dd623c97817f72e5a1c7c7055495ab44308/app/src/main/res/layout/activity_about.xml#L207) 

2) [translators ](https://github.com/ArunPrasanth-V/AmazeFileManager/blob/a7da8dd623c97817f72e5a1c7c7055495ab44308/app/src/main/res/layout/activity_about.xml#L734),[contributors](https://github.com/ArunPrasanth-V/AmazeFileManager/blob/a7da8dd623c97817f72e5a1c7c7055495ab44308/app/src/main/res/layout/activity_about.xml#L1224) to RecyclerView. 

if i'm worng correct me!

**Comment by VishnuSanal** [comment]
> 1. In line number [63](https://github.com/ArunPrasanth-V/AmazeFileManager/blob/a7da8dd623c97817f72e5a1c7c7055495ab44308/app/src/main/res/layout/activity_about.xml#L63) only

Need not replace the root `LinearLayout` (L#63), IMO - because it only contains elements that should be arranged linearly. 

> (or) i need change [ 76](https://github.com/ArunPrasanth-V/AmazeFileManager/blob/a7da8dd623c97817f72e5a1c7c7055495ab44308/app/src/main/res/layout/activity_about.xml#L76), [authors (207)](https://github.com/ArunPrasanth-V/AmazeFileManager/blob/a7da8dd623c97817f72e5a1c7c7055495ab44308/app/src/main/res/layout/activity_about.xml#L207), [translators ](https://github.com/ArunPrasanth-V/AmazeFileManager/blob/a7da8dd623c97817f72e5a1c7c7055495ab44308/app/src/main/res/layout/activity_about.xml#L734),[contributors](https://github.com/ArunPrasanth-V/AmazeFileManager/blob/a7da8dd623c97817f72e5a1c7c7055495ab44308/app/src/main/res/layout/activity_about.xml#L1224) to RecyclerView.

Yes, you'd need to replace every `LinearLayout`s inside those five `CardView`s to `ConstraintLayout`s

**Comment by MohitMandalia** [comment]
Is this issue open for contribution ??

**Comment by VishnuSanal** [comment]
@ArunPrasanth-V Any updates here?

**Comment by ArunPrasanth-V** [comment]
@VishnuSanal I have changed LinearLayout to ConstraintLayout. Have a look at it!

**Comment by ArunPrasanth-V** [comment]
@VishnuSanal I couldn't figure out where the view was overdrawn.

**Comment by VishnuSanal** [comment]
> @VishnuSanal I couldn't figure out where the view was overdrawn.

@ArunPrasanth-V I'll review your PR & give you my suggestions there, in a while. :)

**Comment by VishnuSanal** [comment]
@ArunPrasanth-V 

> @VishnuSanal I couldn't figure out where the view was overdrawn.

Okay, as pointed out early, changes on #3109 does not reduce any overdraw. The intention of this issue was to necessarily remove the nested layouts and replace them with a `ConstraintLayout` at the root.

Please refer to the tageed comment from this thread for reference. :)

> 
> > 1. In line number [63](https://github.com/ArunPrasanth-V/AmazeFileManager/blob/a7da8dd623c97817f72e5a1c7c7055495ab44308/app/src/main/res/layout/activity_about.xml#L63) only
> 
> Need not replace the root `LinearLayout` (L#63), IMO - because it only contains elements that should be arranged linearly.
> 
> > (or) i need change [ 76](https://github.com/ArunPrasanth-V/AmazeFileManager/blob/a7da8dd623c97817f72e5a1c7c7055495ab44308/app/src/main/res/layout/activity_about.xml#L76), [authors (207)](https://github.com/ArunPrasanth-V/AmazeFileManager/blob/a7da8dd623c97817f72e5a1c7c7055495ab44308/app/src/main/res/layout/activity_about.xml#L207), [translators ](https://github.com/ArunPrasanth-V/AmazeFileManager/blob/a7da8dd623c97817f72e5a1c7c7055495ab44308/app/src/main/res/layout/activity_about.xml#L734),[contributors](https://github.com/ArunPrasanth-V/AmazeFileManager/blob/a7da8dd623c97817f72e5a1c7c7055495ab44308/app/src/main/res/layout/activity_about.xml#L1224) to RecyclerView.
> 
> Yes, you'd need to replace every `LinearLayout`s inside those five `CardView`s to `ConstraintLayout`s

**Comment by VishnuSanal** [comment]
@ArunPrasanth-V Are you still working on this?

**Comment by bimalkaf** [comment]
Hi,  I am new to open source contribution, But i know development.
Can i start working on this as this is good first issue?

**Comment by hammadrfq3** [comment]
Is it still open to work? Can i contribute to it?

**Comment by hammadrfq3** [comment]
Overdraw issue has been fixed in activity_about.xml. Should i open pull request now?
